domain/src/main/java/nl/tehdreamteam/se42/domain/message/TextMessage.java

```diff
@@ -0,0 +1,39 @@
+package nl.tehdreamteam.se42.domain.message;
+
+import nl.tehdreamteam.se42.domain.Message;
+import nl.tehdreamteam.se42.domain.User;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+
+/**
+ * @author Oscar de Leeuw
+ */
+@Entity
+public class TextMessage extends Message<String> {
+
+    @Column(name = "text")
+    private String textContent;
+
+    protected TextMessage() {
+    }
+
+    public TextMessage(String content, User sender) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AUndocumentedApi)